### PR TITLE
Early draft of first alumni outreach email

### PR DESCRIPTION
**Pull request by kerrizor**
@theutahkate and I hacked this up. Thoughts @OSFeels/organizers ?

### CODE REVIEW

File: outreach/announcement-to-alumni.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Hello friend! Did you miss us? We missed you..
+
+[*awesome pic*]
+
+We're back, and we're super excited to announce that Open Source & Feelings will be on **July 22-23, 2016** at the same venue as last year, Broadway Performance Hall, 1625 Broadway Ave in Seattle, Washington.
```
Comment (jnf) [inline outreach/announcement-to-alumni.md]: Maybe switch to a colon at  `...the same venue as last year**:** the Broadway Performance Hall, 1625...` Maybe link to the [BPH](http://wp.seattlecentral.edu/broadway-performance-hall/) too?

File: outreach/announcement-to-alumni.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+Just kidding! We will tweet all of announcements so loud and so often, we will make angels weep. Twitter angels. All you have to do to stay informed is to keep an eye on the [@osfeels Twitter account](http://twitter.com/osfeels), and you will be rewarded with information, exuberance, and adorable GIFs.
+
+One final bit of news. It is very important to us that Open Source and Feelings be as accessible a conference as we can make it. We are committed to reducing barriers to attending and/or speaking that many people face - like the cost of taking time away from work, childcare, travel, or the cost of a ticket. Last year we worked very hard at keeping our General Admission ticket price as low as possible, and giving scholarships to anyone who asked for them. We couldn't have done this without the strong support of [our sponsors](http://www.osfeels.com/sponsors/) as well as the 33 individuals who were able to purchase a higher priced ticket. We are deeply grateful for your support!
```
Comment (jnf) [inline outreach/announcement-to-alumni.md]: - I love the _We are committed..._ sentence. It's something we should pull into a mission/values statement on the main website.
- Verb tense mismatching. Maybe update to something like: `Last year, we worked very hard to keep ticket prices low, and to offset costs via scholarships and honorariums.`
- ...the 33 individuals who purchased higher-priced _Community Support_ tickets. We are deeply grateful for your support!

File: outreach/announcement-to-alumni.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+And...we can do better. In that spirit, we've [opened ticket sales for 2016](https://ti.to/osfeels/open-source-feelings-2016) with a small batch of "Early Supporter" tickets. These tickets are priced higher than General Admission tickets, which will go on sale in a couple months. You will receive the same exact privileges and benefits as with a GA ticket, and we will still have plenty of GA tickets available when we open sales.
```
Comment (jnf) [inline outreach/announcement-to-alumni.md]: Artisanal organic small batch tickets? ^_^

There's a lot of information in this paragraph and the next, but I think it's out of order. Let's pull the info about upcoming GA tickets into their own sentences, after we talk about _Early Supporter_. Something like:

> An _Early Supporter_ ticket, like last year's _Community Support_, is priced higher than its General Admission counterpart. The extra money from these ticket sales goes directly into our travel assistance fund. Building this fund early allows us to offset costs of attendance for speakers through travel assistance, housing stipends, and honorariums.

> Plentiful General Admission tickets will be available in a couple of months. We're planning a price point similar to last year's conference. We'll tweet voraciously and send another email when they go on sale.

**Comment by kerrizor** [comment]
Thanks everyone! @theutahkate any more thoughts before we send it out? ..or awesome image ideas? :D

**Comment by theutahkate** [comment]
These comments are great! I think it's ready to go. I'm not having any awesome image ideas, sadly. But I think it'll be okay.

**Pull request merged**